css/datagrid/datagrid.css

```diff
@@ -296,7 +491,10 @@ th.highcharts-datagrid-column-sortable > .highcharts-datagrid-column-sortable-ic
     height: 100%;
     outline: none;
     float: left;
-    font-size: 1em;
+    font: var(--hcdg-font);
+    box-shadow: inset 0 0 0 1px #0000ed;
```

Comment:
Could it be an idea to move this `box-shadow` to the `<td>` element, for consistency? Basically change the color of the one already there, instead of adding an additional one? Then the input is completely unstyled, and we have freedom to do more with it later if needed.